src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -205,7 +201,7 @@ internal static string DefaultPackage
             {
                 _context = new NTAuthentication(isServer, package, credential, servicePrincipalName, flags, channelBinding);
             }
-            catch (Win32Exception e)
+            catch (Exception e)
```

Comment:
If NTAuthentication constructor on unix throws an exception that is not  Win32Exception.